birdhouse/config/magpie/permissions.cfg.template

```diff
@@ -18,7 +18,7 @@ permissions:
   - service: api
     resource: /api
     permission: read
-    user: anonymous
+    group: anonymous
```

Comment:
What is this api service?  I have a vague memory in the other magpie upgrade PR we are going to remove it?

Reply:
`ServiceAPI` is the service that converts `/some/sub/route` into resources: 
```
some:
  sub: 
    route:
```
with permissions applicable on each path element.

It is definitely not being removed. It is the 2nd most used after `ServiceWPS`. Both will probably be combined to form a "super" `ServiceADES` (https://github.com/Ouranosinc/Magpie/issues/360) that combines all WPS-1/REST process interfaces.

Reply:
Oh ok thanks.  I mixed up I guess.